api.py

```
@@ -277,3 +297,46 @@ async def api_athlete_pb_table(
+    swimmers = db.execute(stmt).scalars().all()
+
+    return swimmers
+
+@router.get(
+    "/athlete",
+    response_class=JSONResponse,
+    summary="Returns json data for specific athlete",
+    description="TODO: WIP"
+)
+async def api_athlete_endpoint(
+        request: Request,
```

Ruff (0.14.5)

Unused function argument: `request`

(ARG001)